### DISCUSSION TITLE
Use of third party swift packages in ios plugin

### DISCUSSION THREAD

**Discussion by justinpathrose** [ionic-team/capacitor#6661] [Q&A]
We are developing a capacitor plugin for inttegrating salesforce marketing cloud push. For this we are having to include SFMC sdk which is a swift package. We are now adding the dependency after running cap sync and it works. But the depency is removed every time cap sync is run. 
Any idea on how to embed this depedency (swift package) in plugin code?

**Comment by jonathankehler** [comment]
@justinpathrose I'm trying something similar. To which Xcode project did you add the dependency after running cap sync?
I have tried adding it to the main xcode project (not the plugin one). But for some reason swift files in my plugins cannot import the package. But swift files elsewhere in the project can. Any suggestions?
